### ISSUE TITLE
Set webhook for Fedora Weblate

### ISSUE THREAD

**Issue by rffontenelle** [libguestfs/libguestfs#151]
The following [alert is currently shown in Fedora Weblate](https://translate.fedoraproject.org/projects/libguestfs/libguestfs-docs-master/?alerts=1#alerts):

> Repository outdated.
> 
> The VCS repository is not up to date with the upstream one. Please merge changes manually or set up hooks to automate this.
> [\[Documentation\]](https://docs.weblate.org/en/weblate-5.6.2/admin/continuous.html#update-vcs)

Can you please set up a webhook, so the translation strings are continuously updated?

**Comment by rwmjones** [comment]
I don't understand what to do.  Also at one point this was working, then it stopped working, so I guess this has already been set up but broke for whatever reason.

**Comment by rffontenelle** [comment]
The "Documentation" links actually is very direct: it shows how to set a webhook between your repository and the Weblate instance. Direct link to Github webhook info: https://docs.weblate.org/en/weblate-5.6.2/admin/continuous.html#github-setup

Please notice the translations are not broken, but they are not automatically receiving changes from this repository. I (with my Weblate admin privileges) could run a Update button that would fetch changes from your repository, but it would be better if a webhook is set to send changes without needing someone to do it manually.

**Comment by rffontenelle** [comment]
@rwmjones Were you able to check whether the webhook ~`https://hosted.weblate.org/hooks/github/`~ `https://translate.fedoraproject.org/hooks/github/` is set in this repository? Here's an update URL for docs: https://docs.weblate.org/en/weblate-2026.6.1/admin/code-hosting.html#github-notifications

EDIT: fixed the URL

**Comment by rwmjones** [comment]
I notice the webhook URL is different.  For https://github.com/libguestfs/virt-v2v/issues/164 you suggested using `https://translate.fedoraproject.org/hooks/github/` which is different from above and the one suggested in the weblate docs.  Which is correct?

**Comment by rffontenelle** [comment]
You're correct. The one I provided is for hosted weblate, which I copied but didn't update the base url. I fixed the URL to avoid further confusion.

**Comment by rwmjones** [comment]
This one should be done now.

**Comment by rwmjones** [comment]
Closing as this appears to be working fine.
